jib-cli/src/test/java/com/google/cloud/tools/jib/cli/buildfile/TestObjectMapper.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.cli.buildfile;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+/**
+ * Helper class to create object mappers for all tests that match what we use in the actual code.
+ */
+public class TestObjectMapper {
+
+  public static ObjectMapper newObjectMapper() {
+    // TODO: in future, create this mapper from the mapper creation in the main program
```

Comment:
I have trouble understanding what this is saying. (Specifically, I don't get what "mapper creation" and "main program" mean here.) And why would one want to do so?

Reply:
Valid question, for some restrictions/features, I believe you can only apply them on the mapper during creation and not on the class with an annotation.

I guess we still wouldn't need this test mapper, could probably just call the mapper creator from the tests directly.